lib/node.js

```diff
@@ -935,6 +935,7 @@ declare class http$ServerResponse extends stream$Writable {
 declare module "http" {
   declare class Server extends net$Server {
     listen(port: number, hostname?: string, backlog?: number, callback?: Function): Server;
+    listen(port: number, callback?: Function): Server;
```

Comment:
Thanks for reply. Below example causes flow to complai with "Error:(23, 1) _Flow: call of method `listen`. Function cannot be called on any member of intersection type intersection_ ":
```js
server.listen(3000, function () {
  console.log('Server is listening on port 3000')
})
```
So the signature you suggested doesn't match.
If I add missing parameters -- no errors then:
```js
server.listen(3000, '', 0, function () {
  console.log('Server is listening on port 3000')
})
```
Flow version 0.43.1